library/src/main/java/com/squareup/timessquare/CalendarPickerView.java

```diff
@@ -6,14 +6,16 @@
 import android.content.res.Resources;
 import android.content.res.TypedArray;
 import android.graphics.Typeface;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
 import android.util.AttributeSet;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-import android.widget.BaseAdapter;
-import android.widget.ListView;
 import android.widget.Toast;
+
```

Comment:
revert all these whitespace changes in the imports.  if you use https://github.com/square/java-code-styles IntelliJ/AndroidStudio will format these correctly for you.